plugins/hydra_ray_aws_launcher/hydra_plugins/hydra_ray_aws_launcher/conf/hydra/launcher/ray_aws.yaml

```diff
@@ -0,0 +1,149 @@
+hydra:
+  launcher:
+    cls: hydra_plugins.hydra_ray_aws_launcher.ray_aws_launcher.RayAWSLauncher
+    params:
+      cluster_config:
+        # this maps to a Ray cluter config, e.g:
+        # https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-full.yaml
+        # An unique identifier for the head node and workers of this cluster.
+        cluster_name: default
+
+        # The minimum number of workers nodes to launch in addition to the head
+        # node. This number should be >= 0.
+        min_workers: 0
+
+        # The maximum number of workers nodes to launch in addition to the head
+        # node. This takes precedence over min_workers.
+        max_workers: 2
+
+        # The initial number of worker nodes to launch in addition to the head
+        # node. When the cluster is first brought up (or when it is refreshed with a
+        # subsequent `ray up`) this number of nodes will be started.
+        initial_workers: 0
+
+        # Whether or not to autoscale aggressively. If this is enabled, if at any point
+        #   we would start more workers, we start at least enough to bring us to
+        #   initial_workers.
+        autoscaling_mode: default
+
+        # This executes all commands on all nodes in the docker container,
+        # and opens all the necessary ports to support the Ray cluster.
+        # Empty string means disabled.
+        docker:
+            image: "" # e.g., tensorflow/tensorflow:1.5.0-py3
+            container_name: "" # e.g. ray_docker
+            # If true, pulls latest version of image. Otherwise, `docker run` will only pull the image
+            # if no cached version is present.
+            pull_before_run: True
+            run_options: []  # Extra options to pass into "docker run"
+
+            # Example of running a GPU head with CPU workers
+            # head_image: "tensorflow/tensorflow:1.13.1-py3"
+            # head_run_options:
+            #     - --runtime=nvidia
+
+            # worker_image: "ubuntu:18.04"
+            # worker_run_options: []
```

Comment:
I copied this file directly from Ray's example which has all the advanced options possible. How about instead of showing& commenting them out here we just not show them. Advanced users will have a better idea of what they are configuring, I'm concerned by adding defaults for advanced options here we are making it harder for users to start with something that just works.